map_view/display-device-location-with-autopan-modes/README.metadata.json

```
@@ -0,0 +1,37 @@
+{
+    "category": "Map",
+    "description": "Display your current position on the map, as well as switch between different types of auto pan modes.",
+    "ignore": false,
+    "images": [
+        "DisplayDeviceLocation.png"
+    ],
+    "keywords": [
+        "GPS",
+        "compass",
+        "location",
+        "map",
+        "mobile",
+        "navigation",
+        "ArcGISMap",
+        "LocationDisplay",
+        "LocationDisplay.AutoPanMode",
+        "MapView",
+        "SimulatedLocationDataSource",
+        "SimulationParameters"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/display-a-map.htm"
+    ],
```

eek definitely not :D 
```suggestion
    "redirect_from":  "",
```